plugins/haproxy.yaml

```diff
@@ -178,8 +178,8 @@ pipeline:
     to: '$attributes["frontend_ssl_ciphers"]'
   - id: method_move
     type: move
-    if: '$attributes.method != nil'
```

Comment:
i don't think this is right?

Reply:
Im not sure what that is, this is how it looks
```yaml
  - id: method_move
    type: move
    if: '$body.method != nil'
    from: '$body.method'
    to: '$attributes["method"]'
```